project-reports/2023/2023-Q1-Hyperledger-Indy.md

```diff
@@ -0,0 +1,162 @@
+to the new Hyperledger AnonCreds project and the [anoncreds-rs] repository. In
+the shared components migration from the Indy SDK talked about above, the
+[anoncreds-rs] AnonCreds implementation will be used. A pair of Indy AnonCreds
+Methods will remain, one based on the existing Indy AnonCreds objects
+identifiers, and one based on the new "did:indy" object identifiers. This change
+will simplify the transition to "did:indy".
+
+[anoncreds-rs]: https://github.com/hyperledger/anoncreds-rs
+
+Per the [Indy Activity Dashboard (2022-10 to 2022-12)], there were 100 commits
+(almost double from last quarter) from 13 contributors (up one).
+
+[Indy Activity Dashboard (2022-10 to 2022-12)]: https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Findy/dashboard;subTab=technical?time=%7B%22from%22:%222022-10-01T07:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222022-12-31T07:00:00.000Z%22%7D
+
+## Questions/Issues for the TSC
+
+### Issues from previous reports
+
+#### Build Pipelines
+
+**Update** : The Ubuntu upgrade is complete, producing releases and
+compatibility with the Ubuntu 16.04 instance has been tested. To be removed from
+the next report.
+
+#### **Diversity of Contributor Community**
+
+**No Update** : Little change this quarter in contributor community. Lots
+of interest, but core maintainers continue to do most of the work.
+
+## Releases
+
+- indy-node (Ubuntu 20.04) – v1.13.2-rc4
+
+## Overall Activity in the Past Quarter
+
+In the past quarter (as in the previous quarter), ledger code
+development focused on code management – upgrading the Indy Node and
+Plenum CI/CD pipeline and upgrading Indy Node to run on Ubuntu 20.04.
+A particular focus was on the [indy-test-automation] pipeline, and
+enabling its use for both Indy Node releases and for the downstream
+Sovrin Node releases.
+
+Much activity occurred in extracting the AnonCreds Rust implementation
+from the [indy-shared-rs] repository to the standalone [anoncreds-rs]
+repository.
+
+## Current Plans
+
+With the new CI/CD Indy Node and Plenum pipelines complete, and the
+Ubuntu 20.04 indy-node upgrade available, the core maintainers are
+focused on their downstream releases.
+
+The Aries Frameworks are moving to [indy-vdr] and [aries-askar] and away from
+[indy-sdk]. As that happens, adjustments and additions will be made to the newer
+repositories. The high priority work on Hyperledger AnonCreds implementation
+will also drive that effort.
+
+A focus on pushing the "did:indy" DID Method is enabled by the upgrade of Indy
+Node to the new Ubuntu 20.04-based releases.
+
+## Maintainer Diversity
+
+The bi-weekly Indy Contributors call continues to be the medium by which
+maintainers coordinate work, discuss critical issues to the Indy
+codebase, and agree on larger changes. Topics and attendance has
+increased recently, with more and more interested parties showing up,
+and new contributors weighing in on the conversations.
```

Comment:
For this section, the TOC is interested in understanding what organizations the maintainers are from. Do they all fall under a single org? Is there diversity in the maintainers?

Reply:
Same as with previous comment -- a leftover from previous reports. I'll see what additional information can be added next quarter. Maintainers are listed in each repo, but getting a consolidated list across repos will take a script of some kind.

Perhaps each project should have a maintainer team hierarchy -- a "project-name" maintainers team with all maintainers across all repos, and a per repo maintainers team with members all a part of the project team.

Reply:
> Perhaps each project should have a maintainer team hierarchy -- a "project-name" maintainers team with all maintainers across all repos, and a per repo maintainers team with members all a part of the project team.

@swcurran Sounds like a good idea to me. The GitHub REST API can be used to traverse organization repos so a script that uses that could be nice.

Reply:
> Same as with previous comment -- a leftover from previous reports. I'll see what additional information can be added next quarter. Maintainers are listed in each repo, but getting a consolidated list across repos will take a script of some kind.
> 
> Perhaps each project should have a maintainer team hierarchy -- a "project-name" maintainers team with all maintainers across all repos, and a per repo maintainers team with members all a part of the project team.

@swcurran talk to me on discord